graphql_compiler/compiler/emit_sql.py

```diff
@@ -167,6 +202,8 @@ def backtrack(self, previous_location):
 
     def traverse(self, vertex_field, optional):
         """Execute a Traverse Block."""
+        if self._in_fold:
+            raise NotImplementedError("Traversals inside a fold are not implemented yet.")
```

Comment:
I'm worried implementing this will cause a major redesign of the state we store and update for fold. Do you have an idea of what is involved in implementing it?